wagtailio/project_styleguide/templates/patterns/components/streamfields/cards/card_block.html

```diff
@@ -1,34 +1,14 @@
-{% comment "temporary styles for testing." %}
-Remove once the styling for these css classes is defined globally
-{% endcomment %}
-<style>
-    .eggshell {
-        fill: #B3E7E8;
-    }
+<div class="card">
+    <span>
+        {% include "patterns/components/meta/meta.html" with classes="card__meta" %}
```

Comment:
Question: In Builder there is an option for a `Meta Icon` - does this need to be passed to the `meta` component?